### ISSUE TITLE
qwen2.5vl train loss is super lower than qwen2vl but performance is wired

### ISSUE THREAD

**Issue by KaiWU5** [QwenLM/Qwen2.5-VL#766]
I use the same codebase transformers train qwen2.5vl and qwen2vl. 

For qwen2vl the loss is normal around 2.0, but for qwen2.5vl the loss is super low around 0.4. But the qwen2.5vl performance is weird which cannot output a normal sentence.  Does any one have the same issue?

**Comment by Elenore1997** [comment]
I use Llama-Factory to fine-tune qwen2.5vl with the same dataset as qwen2vl, but got lots of empty output when inference the trained model.

**Comment by KaiWU5** [comment]
> I use Llama-Factory to fine-tune qwen2.5vl with the same dataset as qwen2vl, but got lots of empty output when inference the trained model.

meeeeee to! Nearly without changing fine-tuning code, the qwen2.5vl sft results is a lot worse than qwen2vl. Please share some experience @ShuaiBai623

**Comment by Elenore1997** [comment]
> > I use Llama-Factory to fine-tune qwen2.5vl with the same dataset as qwen2vl, but got lots of empty output when inference the trained model.
> 
> meeeeee to! Nearly without changing fine-tuning code, the qwen2.5vl sft results is a lot worse than qwen2vl. Please share some experience [@ShuaiBai623](https://github.com/ShuaiBai623)

About empty output when inferencing: probably related to batch inference (bs > 1) https://github.com/QwenLM/Qwen2.5-VL/issues/759

**Comment by KaiWU5** [comment]
I even simultaneously train Qwen2VL and Qwen2.5VL to compare.

Observations:
The loss for Qwen2.5VL and Qwen2VL is similar.
Finetuning on LLavanext like sft dataset, Qwen2.5VL is a lot worse than Qwen2VL (on opencompass dataset got 40 vs 50 repectively for 3b model).
When I further test on open domain questions on finetined Qwen2.5vl exhibits more repeating or disorder output than Qwen2VL.
Thoughts:
Before fine-tuning Qwen2VL with transformers, there are bugs like 3d-RPOE actually use 2d-RPOE in transformers. I check the implementation the rope seems right.
Could you share some thoughts on the difference in architecture of Qwen2VL and Qwen2.5VL so that I can check. (There is a 4D attention concept in Qwen2.5VL which I didn't find. )
